src/index.html

```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Todo-list</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@100;300;400;500;700&display=swap"
+      rel="stylesheet"
+    />
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css"
+    />
+  </head>
+  <body>
+    <div class="todo-list">
+      <div class="title">
+        <p>Crysty's todo</p>
+        <i class="fa fa-refresh"></i>
+      </div>
+      <div class="input-section">
+        <input type="text" placeholder="add to your list..." id="todo-input" />
+      </div>
+      <ul class="lists"></ul>
+      <button type="button" id="btn">Clear all completed</button>
+    </div>
+    <!-- Background -->
+    <div class="context"></div>
+
+    <div class="area">
+      <ul class="circles">
+        <li></li>
+        <li></li>
+        <li></li>
+        <li></li>
+        <li></li>
+        <li></li>
```

Comment:
I would be better to use sematic tags as improve accessibility of your project. You could use main to contain main to contain the whole body and add sections where appropriate